src/crud.rs

```diff
@@ -71,6 +99,22 @@ macro_rules! crud {
             path.ensure()?;
             let path_hash = path.hash()?;
             create_link(path_hash, entry_hash.clone(), ())?;
+
+            // create a time_path
+            let date: ::chrono::DateTime<::chrono::Utc> = now_date_time()?;
```

Comment:
```suggestion
            let date = now_date_time()?;
```

You can omit the type in a situation like this as it will be inferred